metadash/src/store.js

```diff
@@ -111,6 +111,9 @@ const store = new Vuex.Store({
       _changedConfigs[key] = value
       dispatch('saveConfigs')
     },
+    getConfig ({state, dispatch}, {key}) {
```

Comment:
Hi, This should be a getter, not an action

See: https://vuex.vuejs.org/guide/getters.html#method-style-access

Reply:
fixed in new version